iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkSerDe.java

```diff
@@ -231,6 +234,19 @@ private static boolean ignoreEquipmentAtExport(Identifiable<?> identifiable, Net
                 || identifiable instanceof OverloadManagementSystem && !context.getOptions().isWithAutomationSystems();
     }
 
+    private static boolean isExtensionExcluded(ExtensionSerDe extensionSerDe, ExportOptions options) {
+        if (extensionSerDe == null) {
+            return false;
+        }
+        return options.getIncludedExtensions().filter(Set::isEmpty).isPresent() ||
+                options.getExcludedExtensions()
+                        .map(extensions -> extensions.contains(extensionSerDe.getExtensionName()))
+                        .orElse(false) &&
+                        options.getIncludedExtensions()
+                                .map(extensions -> !extensions.contains(extensionSerDe.getExtensionName()))
+                                .orElse(false);
+    }
```

Comment:
You could use the method `withExtension` here, and inverse the logic:
```suggestion
    private static boolean isExtensionIncluded(ExtensionSerDe extensionSerDe, ExportOptions options) {
        if (extensionSerDe == null) {
            return false;
        }
        return options.withExtension(extensionSerDe.getExtensionName());
    }
```

In that case you have to remove a "!" where this method is used